test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -589,6 +589,38 @@ public void CheckToOtlpLogRecordBodyIsPopulated(bool includeFormattedMessage)
         Assert.Equal("state", otlpLogRecord.Body.StringValue);
     }
 
+    [Fact]
+    public void LogRecordBodyIsExportedWhenUsingBridgeApi()
+    {
+        var logRecords = new List<LogRecord>();
+
+        using (var loggerProvider = Sdk.CreateLoggerProviderBuilder()
+            .ConfigureServices(services =>
+            {
+                services.AddLogging(builder => builder.AddOpenTelemetry(options =>
```

Comment:
curious why this pattern? why not

```csharp
Sdk.CreateLoggerProviderBuilder()
.AddInMemoryExporter(logRecords)
.Build();
```

Reply:
Missed this https://github.com/open-telemetry/opentelemetry-dotnet/pull/5268#discussion_r1468249286. @Yun-Ting Could you please add a comment there on why are using this pattern. 
I think we could consider internalsvisibleto from InMemory Exporter - @CodeBlanch what do you think?

Reply:
Works for me! Updated